jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageTarball.java

```diff
@@ -48,6 +49,7 @@
   private final Image image;
   private final ImageReference imageReference;
   private final ImmutableSet<String> allTargetImageTags;
+  private final Instant creationTime = Instant.EPOCH;
```

Comment:
```suggestion
  private static final Instant TAR_ENTRY_MODIFICATION_TIME = Instant.EPOCH;
```
and move the field up.